### PR TITLE
feat: less visual weight

### PR DESCRIPTION
**Pull request by drdelambre**
now that we're encouraging people to insert more panels in their notebooks, and they're getting introduced to the concept, lets dedicate less visual space to the edge cases of that process. 

query development in notebooks is predominately a top to bottom kind of thing. we want them to start, find something interesting, move on to finding something else interesting, and have a record of that whole exploration. if we think of the notebook as a chronological exploration and not as an artifact, inserting panels between each other becomes an edge case, predominately used for polishing an exploration into an artifact. because we want to focus on that initial exploratory interaction, even though we still need to give the user the affordances to do these actions for later, we dont need to make it a first level interaction.

starts to look a bit like this:
Screenshot 2022-03-11 8 38 58 AM

hover state makes it pop a little more  when you get close to the button:
Screenshot 2022-03-11 8 49 47 AM

### CODE REVIEW

File: src/flows/components/panel/InsertCellButton.tsx [hunk 1]
```diff
@@ -69,6 +66,65 @@ const InsertCellButton: FC<Props> = ({id}) => {
...
+              direction={FlexDirection.Column}
+              alignItems={AlignItems.Stretch}
+              margin={ComponentSize.Small}
+              className="insert-cell-menu"
+            >
+              <p className="insert-cell-menu--title">Insert Panel Here</p>
+              <AddButtons index={index} onInsert={onHide} />
+            </FlexBox>
+          )}
+        />
```
Comment (appletreeisyellow) [inline src/flows/components/panel/InsertCellButton.tsx:123]: seems like popover is not showing up

Comment (appletreeisyellow) [reply src/flows/components/panel/InsertCellButton.tsx:123]: nvm, after i pull in the updated code, it's working now!

Comment (drdelambre) [reply src/flows/components/panel/InsertCellButton.tsx:123]: did you try with the latest code push? i had the targetref attached to the button like it was before, so if you click on the purple button you get the popup, but when the hover state changes the text color like it does, you expect the text to be clickable too. i tried clicking on the text... no popup... so i changed the targetref to include it

**Review by appletreeisyellow — APPROVED** [review]
Looks good to me!

**Review by asalem1 — COMMENTED** [review]
Does this remove the plus sign on the left?

File: src/flows/components/panel/InsertCellButton.tsx (hunk 1 above)
Comment (drdelambre) [reply src/flows/components/panel/InsertCellButton.tsx:123]: lol while i was typing

**Pull request merged**